docs/Apache-HTTP-Server-Logs-Collection.md

```diff
@@ -4,13 +4,16 @@
 
 2. Download and install [OMS Agent for Linux](https://github.com/Microsoft/OMS-Agent-for-Linux) on the machine.
 
-3. If `/etc/opt/microsoft/omsagent/conf/omsagent.d/apache_logs.conf` is not present, move `apache_logs.conf`:
+3. You can get the workspace ID for your machine from the OMS portal after it has onboarded successfully.
+   ![OMSPortalWorkspaceID](pictures/OMSPortalWorkspaceID.PNG?raw=true)
```

Comment:
Also, you should crop the picture so that the primary/secondary keys aren't visible. A workspace ID isn't really a secret, but keys are (I believe).